release.py

```diff
@@ -631,7 +631,7 @@ def select_gpg_key():
     contents = f.read()
 if not user_ok("Going to build and upload mvn artifacts based on these settings:\n" + contents + '\nOK (y/n)?: '):
     fail("Retry again later")
-cmd("Building and uploading archives", "./gradlewAll uploadArchives", cwd=kafka_dir, env=jdk8_env, shell=True)
+cmd("Building and uploading archives", "./gradlewAll publish", cwd=kafka_dir, env=jdk8_env, shell=True)
```

Comment:
the root/`README.md` still use `./gradlewAll uploadArchives`. It would be great to make them consistent.

Reply:
Good catch. I had searched for this, but somehow missed it.